imports/api/server/methods.tests.js

```diff
@@ -0,0 +1,160 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
All tests in this files are about the methods in the file `imports/api/server/methods.js`